fn_ansible/fn_ansible/util/config.py

```diff
@@ -0,0 +1,26 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate a default configuration-file section for fn_ansible"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+   config_data = u"""[fn_ansible]
+[fn_ansible]
+playbook_dir=</full/path/to/your/playbook/directory>
+user_name=<USERNAME-OF-HOSTS>
+root_password=<PASSWORD-OF-HOSTS>
+hosts_path=</full/path/of/your/inventory/file>
+playbook_become_method=<SUPER-USER-METHOD e.g. sudo>
```

Comment:
we may want to recommend that a company use our res-keyring capability to hide passwords:
root_password=^root_password.

see https://github.com/ibmresilient/resilient-reference/raw/master/developer_guides/Resilient%20IRP%20Function%20Developer%20Guide.pdf

Reply:
Done!